packages/x-adapter-platform/src/mappers/request/tagging-request.mapper.ts

```diff
@@ -0,0 +1,10 @@
+import { createMutableSchema, Schema, schemaMapperFactory } from '@empathyco/x-adapter-next';
```

Comment:
Nope, I've created it because we need it. 

The Tagging I think is quite special case since we only have to create the url coming from the `TaggingRequest`. This `TaggingRequest` is an object containing two fields: `params` and `url`. 

If we don't provide a request mapper then the default `identityMapper` will be used and the params and url fields from the tagging request will be mapped as query parameters. The main purpose of this `tagging-request.mapper` is to prevent that and make sure the `TaggingRequest` is correct. Otherwise we will have the right url with some query parameters appended like 
`...params=%5Bobject+Object%5D&url=https%3A%2F%2Fapi.staging.empathy.co%2Ftagging%2Fv1%2Ftrack%2Fempathy%2Fclick`

Reply:
You're totally right. Very well explained too, thanks!